packages/purpose-process/test/testCreatePurpose.ts

```diff
@@ -0,0 +1,333 @@
+      await writeInReadmodel(agreementEservice, agreements);
+      await writeInReadmodel(toReadModelEService(eService), eservices);
+
+      expect(
+        purposeService.createPurpose(
+          seed,
+          unsafeBrandId(purposeSeed.consumerId),
+          generateId()
+        )
+      ).rejects.toThrowError(tenantKindNotFound(tenantWithoutKind.id));
+    });
+    it("should throw tenantNotFound", async () => {
+      expect(
+        purposeService.createPurpose(
+          purposeSeed,
+          unsafeBrandId(purposeSeed.consumerId),
+          generateId()
+        )
+      ).rejects.toThrowError(tenantNotFound(tenant.id));
+    });
+    it("should throw agreementNotFound", async () => {
+      const descriptor: Descriptor = {
+        ...descriptor1,
+        id: generateId(),
+      };
+
+      const eService: EService = {
+        ...eService1,
+        producerId: tenant.id,
+        id: generateId(),
+        descriptors: [descriptor],
+      };
+
+      const agreement: Agreement = {
+        ...agreementEservice1,
+        id: generateId(),
+        eserviceId: eService.id,
+        descriptorId: descriptor.id,
+        producerId: eService.producerId,
+        consumerId: tenant.id,
+        state: agreementState.draft,
+      };
+
+      const seed: ApiPurposeSeed = {
+        ...purposeSeed,
+        eserviceId: eService.id,
+        consumerId: agreement.consumerId,
+      };
+
+      await writeInReadmodel(tenant, tenants);
+      await writeInReadmodel(agreement, agreements);
+      await writeInReadmodel(toReadModelEService(eService), eservices);
+
+      expect(
+        purposeService.createPurpose(
+          seed,
+          unsafeBrandId(seed.consumerId),
+          generateId()
+        )
+      ).rejects.toThrowError(agreementNotFound(eService.id, tenant.id));
+    });
+    it("should throw organizationIsNotTheConsumer if the requester is not the consumer", async () => {
+      await writeInReadmodel(tenant, tenants);
+      await writeInReadmodel(agreementEservice1, agreements);
+      await writeInReadmodel(toReadModelEService(getMockEService()), eservices);
+
+      const seed: ApiPurposeSeed = {
+        ...purposeSeed,
+        consumerId: generateId(),
+      };
+
+      expect(
+        purposeService.createPurpose(
+          seed,
+          unsafeBrandId(purposeSeed.consumerId),
+          generateId()
+        )
+      ).rejects.toThrowError(
+        organizationIsNotTheConsumer(unsafeBrandId(purposeSeed.consumerId))
+      );
+    });
+    it("should throw riskAnalysisValidationFailed if the purpose has an inactive risk analysis ", async () => {
+      await writeInReadmodel(tenant, tenants);
+      await writeInReadmodel(agreementEservice1, agreements);
+      await writeInReadmodel(toReadModelEService(eService1), eservices);
+
+      const mockInvalidRiskAnalysisForm: RiskAnalysisForm = {
+        ...mockValidRiskAnalysisForm,
+        version: "251",
```

Comment:
```suggestion
        version: "0",
```
In QA we use `0` when we want a non valid riskAnalysis, let's do the same here